config.example.json

```diff
@@ -5,9 +5,7 @@
 		"ClientSecret": "" // https://dev.twitch.tv/
 	},
 	"SQL": {
-		"Host": "",
-		"User": "",
-		"Password": ""
+		"Address": "postgres://user:password@localhost:5432/melonbot"
```

Comment:
Hmm doesn't this depend on how the user has setup postgres?

Reply:
Nah incase theres no password setup, im fairly sure you just omit it. So: `postgres://user@localhost:5432/foo`

Reply:
eShrug ok